sections/featured-collection.liquid

```diff
@@ -62,8 +67,8 @@
           {%- endfor -%}
         {%- endfor -%}
       </ul>
-      {%- if section.settings.collection.all_products_count > 2 and section.settings.swipe_on_mobile and section.settings.products_to_show > 2 -%}
-        <div class="slider-buttons no-js-hidden{% if section.settings.collection.all_products_count < 4 %} medium-hide{% endif %}{% if section.settings.collection.all_products_count < 3 %} small-hide{% endif %}">
+      {%- if show_mobile_slider -%}
+        <div class="slider-buttons no-js-hidden{% if show_mobile_slider == false %} medium-hide small-hide{% endif %}">
```

Comment:
I don't think this `{% if show_mobile_slider == false %}` conditional is needed.